### PR TITLE
replacing pug for html

### LINKED ISSUE #585: Remove pug
## Acceptance criteria
* Change pug to HTML
* Include max line length rule to eslint

## Notes
https://codebeautify.org/pug-to-html-converter

### PR DESCRIPTION
**Pull request by Viterbo**
# Fixes #585

## Description
This PR replaces all pug code for the equivalent in HTML 

## Test scenarios
- Open the [current testing site](https://explorer-test.telos.net/)
- Open [this PR testing link](https://deploy-preview-627--obe-staging.netlify.app/) another browser tab
- Browse whenever you want but do it in both at the same time
- You should not notice any difference

### CODE REVIEW

File: src/components/ProposalAction.vue [hunk 1]
```diff
@@ -161,84 +161,99 @@ export default defineComponent({
...
+                        dense
+                        hide-bottom-space
+                        lazy-rules
+                        label="account"
+                        :error="isAccountError"
+                        :loading="isAccountLoading"
+                        :rules="[value => !!value || 'Account is required']"
+                    />
+                </div>
+                <div v-if="actionOptions.length &gt; 0" class="col-6">
```
Comment (EJSG) [inline src/components/ProposalAction.vue:197]:
```suggestion
                <div v-if="actionOptions.length > 0" class="col-6">
```

File: src/components/TokenSearch.vue [hunk 2]
```diff
@@ -82,36 +82,48 @@ export default defineComponent({
...
+    use-input
+    hide-selected
+    fill-input
+    hide-bottom-space
+    placeholder="token"
+    class="search-input"
+    :loading="isLoading"
+    :model-value="inputValue"
+    :options="options"
+    @input-value="(value) =&gt; inputValue = value"
```
Comment (EJSG) [inline src/components/TokenSearch.vue:99]:
```suggestion
    @input-value="(value) => inputValue = value"
```

**Review by EJSG — CHANGES_REQUESTED** [review]
thank god, i cant wait to have the pug gone :laughing: just 2 issues i found

**Comment by EJSG** [comment]
you can also remove pug loader from `package.json` and `quasar.conf.js` and `jest.config.js`

**Comment by Viterbo** [comment]
@EJSG, thank you for the review!
I fixed the config files and resolve all pending conflicts.

**Review by karynemayer — APPROVED** [review]

**Review by EJSG — APPROVED** [review]
Very nice 💪

**Pull request merged**